### PR TITLE
288 Enhancement to Testlinks 

### PR DESCRIPTION
**Pull request by ahmad-shahid**

### CODE REVIEW

File: TestLinks.js [hunk 1]
```diff
@@ -60,6 +61,10 @@ module.exports = async function testFileLinks() {
     const config = (process.env.DISABLE_PROXY) ? null : {httpsAgent: agent, proxy: false};
 
     let matches = [];
+    let errorCount = 0;
+    let skippedSyntaxUrls = 0;
```
Comment (plup2) [inline TestLinks.js:65]: The only thing:)  Names like `skippedSyntaxUrls` would indicate a variable representing an array of list.  Counters should be named more like `skippedSyntaxUrlCount`  (consistent with "errorCount" :) )

File: package.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
     "grunt-exec": "^3.0.0",
     "grunt-webdriver": "^3.0.0",
     "html-validate": "^6.1.0",
-    "https-proxy-agent": "^5.0.0"
+    "https-proxy-agent": "^5.0.0",
+    "perf_hooks": "0.0.1"
```
Comment (plup2) [inline package.json:44]: perf_hooks is a core module (with node 8.5+)... In theory we shouldn't need to refer to the npm module

File: TestLinks.js (hunk 1 above)
Comment (ahmad-shahid) [reply TestLinks.js]: Fixed!

File: package.json (hunk 2 above)
Comment (ahmad-shahid) [reply package.json]: Removed the reference.

**Pull request merged**